src/14-curated-discovery/01-artworks.ts

```diff
@@ -143,6 +143,24 @@ async function prepareArtworkCollection() {
           },
         },
       },
+      {
+        name: "priceMinMinorUSD",
```

Comment:
Since the `listPriceAmount` can be in non-USD currencies, we should just be careful with any logic that uses both.

Reply:
Yeah, honestly not sure if we even _need_ to keep `listPriceAmount` and `listPriceCurrency`, since we already have  the formatted sale message in `price` for user-facing display, and the new fields for filtering.